utils/slot.mjs

```diff
@@ -6,11 +6,15 @@ let attempts = 0;
 export const watchSlotSent = async (gSlotSent, connection) => {
   while (true) {
     try {
+      let lastSlot = 0;
       const subscriptionId = connection.onSlotUpdate((value) => {
-        if (value.type === "firstShredReceived") {
-          gSlotSent.value = value.slot;
-          gSlotSent.updated_at = Date.now();
-          attempts = 0;
+        if (value.type === "firstShredReceived" || value.type == "completed") {
+          if value.slot > lastSlot {
+            lastSlot = value.slot;
+            gSlotSent.value = value.slot;
```

Comment:
This doesn't quite get at it, I think, because in the event that the type is `completed` that implies that we're on `slot + 1n`.

Here's another try! https://github.com/rpcpool/ping-thing-client/pull/10/commits/35e65bc9e675bfae0ceef833282699dbf04c8c54